spec/factories/course.rb

```diff
@@ -69,6 +69,15 @@
       uuid { SecureRandom.uuid }
     end
 
+    trait :teacher_degree_apprenticeship do
+      apprenticeship
+      full_time
+      description { 'Teacher degree apprenticeship with QTS full time teaching apprenticeship' }
```

Comment:
```suggestion
      description { 'Teaching degree apprenticeship with QTS full time teaching apprenticeship' }
```
Is it Teaching degree apprenticeship or Teacher degree apprenticeship? We are not consistent. it may be too late to change in some  places (eg the programme_type), but we should make an effort where possible.